odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+from odl.contrib.datasets.util import get_data
+
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'cms', 'cms_gray', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images'
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def brain_phantom():
```

Comment:
Also an overall comment: Try to document the size (in pixels) and output type of all the images